docs/3.0rc/develop/task-runners.mdx

```diff
@@ -78,8 +80,10 @@ In this case, you can create [nested flows](/3.0rc/develop/write-flows/#composin
 For example, you can have a flow (in the example below called `multiple_runner_flow`) that runs its tasks locally using the `ThreadPoolTaskRunner`. 
 If you have some tasks that can run more efficiently in parallel on a Dask cluster, you can create a nested flow (such as `dask_nested_flow`) to run those tasks using the `DaskTaskRunner`.
 
+
 ```python
 from prefect import flow, task
+from prefect.futures import wait
```

Comment:
maybe this example should keep the `.wait()` use instead of `wait`? since the mention of the `wait` utility is not directly above this example and ideally the user doesn't need to engage with extra utils / imports to do a normal thing like wait for 2 futures - I think the other use of `wait` is more appropriate because there are many more than 2 futures to await